fxa-auth-db-server/index.js

```diff
@@ -43,14 +43,25 @@ function createServer(db) {
   var api = restify.createServer()
   api.use(restify.bodyParser())
   api.use(restify.queryParser())
-  api.use(bufferize.bufferizeRequest.bind(null, [
-    'uaBrowser',
-    'uaBrowserVersion',
-    'uaOS',
-    'uaOSVersion',
-    'uaDeviceType',
-    'unblockCode'
-  ]))
+  api.use(bufferize.bufferizeRequest.bind(null, {
+    // These are all the different params that we handle as binary Buffers,
+    // but are passed into the API as hex strings.
+    'authKey': true,
+    'authSalt': true,
+    'data': true,
+    'deviceId': true,
+    'emailCode': true,
+    'id': true,
+    'kA': true,
+    'keyBundle': true,
+    'passCode': true,
+    'sessionTokenId': true,
+    'tokenId': true,
+    'tokenVerificationId': true,
+    'uid': true,
+    'verifyHash': true,
+    'wrapWrapKb': true,
```

Comment:
My problem is I don't code regularly enough for the es6isms to be habit yet, so I appreciate the reminders to keep moving in that direction.